### DISCUSSION TITLE
Rewards for the top bug reporters December 2021

### DISCUSSION THREAD

**Discussion by Valsery** [chromiecraft/chromiecraft#2775] [General]
This is the number of VALID bug reports per every contributor that have been validated in December 2021.
When a bug is validated, it gets the Linked [AC] label.

If your bug hasn't been validated yet, don't worry: it will be counted in a later month. Invalid bug reports will of course not be counted.

Source: https://www.chromiecraft.com/top-contributors-of-decembe-2021/

The people mentioned here: in order to receive your Chromie points, please send me a private message on Discord (Gn#6416) with the following:

1. your GitHub username
2. your game account username
3. your game account email

Remember to give us a Star on our main GitHub repo: https://github.com/azerothcore/azerothcore-wotlk

@ishidinho – 25
@Elexias – 13
@baranzadeoglu – 12
@j-hdt – 4
@godnesso – 4
@cachocarne – 4
@Orixim – 4
@creativemonkeyz – 3
@claudiodfc – 3
@Plan414 – 3
@Lucika77 – 3
@kuzaan – 2
@Venom0fGod – 2
@Stump876 – 2
@NellaTomie – 2
@Luckdemon2 – 2
@Hihilist – 2
@phonkala – 1
@m0ll3art – 1
@keldinx – 1
@jeskej – 1
@jburkettstudio – 1
@ingerash4ever – 1
@eTzmNcbkrng – 1
@chaoticbeginning – 1
@chaossonic – 1
@bonfax4 – 1
@balazskvr – 1
@Tokelin – 1
@The-Eyeful – 1
@Szudro718 – 1
@Riveir – 1
@Plasmashield – 1
@Nimbux – 1
@Kike457 – 1
@Joy-ccraft – 1
@Josarth – 1
@HasieEST – 1
@Foodlenz – 1
@Erregwyn – 1
@CMHWoody – 1
@Bredley – 1
@Arhkadian – 1

**Comment by FrancescoBorzi** [comment]
For people receiving emails as notification, as the post was edited, you need to contact: Gn#6416

@ishidinho – 25
@Elexias – 13
@baranzadeoglu – 12
@j-hdt – 4
@godnesso – 4
@cachocarne – 4
@Orixim – 4
@creativemonkeyz – 3
@claudiodfc – 3
@Plan414 – 3
@Lucika77 – 3
@kuzaan – 2
@Venom0fGod – 2
@Stump876 – 2
@NellaTomie – 2
@Luckdemon2 – 2
@Hihilist – 2
@phonkala – 1
@m0ll3art – 1
@keldinx – 1
@jeskej – 1
@jburkettstudio – 1
@ingerash4ever – 1
@eTzmNcbkrng – 1
@chaoticbeginning – 1
@chaossonic – 1
@bonfax4 – 1
@balazskvr – 1
@Tokelin – 1
@The-Eyeful – 1
@Szudro718 – 1
@Riveir – 1
@Plasmashield – 1
@Nimbux – 1
@Kike457 – 1
@Joy-ccraft – 1
@Josarth – 1
@HasieEST – 1
@Foodlenz – 1
@Erregwyn – 1
@CMHWoody – 1
@Bredley – 1
@Arhkadian – 1
